### ISSUE TITLE
Can i have your permission to translate your article into Chinese ?

### ISSUE THREAD

**Issue by hanxiaomax** [g1thubhub/g1thubhub.github.io#1]
https://g1thubhub.github.io/data-structure-zoo.html
this is a very good post, hope Chinese developer can enjoy it too.

**Comment by g1thubhub** [comment]
Sure, we can work together on it, there's still stuff I haven't published and wanted to include

**Comment by hanxiaomax** [comment]
thank you for your reply. About the unpublished part , what's your plan ? could I start to translate the article now ?

**Comment by g1thubhub** [comment]
Hey,

Sorry, have been fairly busy recently jobwise.

There is an unpublished part with code that actually benchmarks all collections mentioned, similar to this post that I published last week https://g1thubhub.github.io/benchmarking.html

Do you want to put the translation on a Chinese blogging platform? We could work together. Concerning the benchmarks, I'm thinking about doing this on Google cloud platform or Amazon but I have to register a new account first

**Comment by hanxiaomax** [comment]
yes, we'll put the translation on our [web site](http://www.jobbole.com/)  and social networks (weibo or wechat) . many developers will read it. 

Since I'm not very familier with the bigdata and their benchmarking so I think I will go ahead with this post (https://g1thubhub.github.io/data-structure-zoo.html) and asking people in our group to see if someone wanna join us.

**Comment by g1thubhub** [comment]
Oki, just include a reference to the original English article, then I also include a reference to the Chinese version. This article will continually expand as I have a few more structures that I want to include, we should also cover Python so maybe we can work together? I also intend to run the benchmarks and unpublished section/code soon...
